src/homeworks/homework9/sum/IncorrectNumberException.java

```diff
@@ -0,0 +1,7 @@
+package homeworks.homework9.sum;
+
+public class IncorrectNumberException extends RuntimeException {
+    IncorrectNumberException() {
+        System.out.println("You enter invalid number!");
```

Comment:
It's better to use super constructor
```suggestion
       supper("You enter invalid number !")
```